RUFAS/biophysical/animal/herd_manager.py

```
@@ -831,6 +835,19 @@ def initialize_pens(
             housing_type = pen_data.get("housing_type", "")
             pen_type = pen_data.get("pen_type", "")
             max_stocking_density = pen_data.get("max_stocking_density", 0.0)
+            minutes_away_for_milking = pen_data.get("minutes_away_for_milking", 0.0)
+            parlor_stream_assignment = pen_data.get("parlor_stream_assignment", None)
+            manure_streams = pen_data.get(
+                "manure_streams",
+                [
+                    {
+                        "stream_name": "general_pen",
+                        "bedding_name": "sand",
+                        "stream_proportion": 1.0,
+                        "first_processor": "general_handler",
+                    }
+                ],
```

This default could be potentially problematic, especially for "first_processor." If we set the user input for "manure_streams" to be nullable and the user doesn't specify a value for it, then the default value here could lead to unexpected behavior. Otherwise, if we make it non-nullable, then we should expect a value for this variable, so I think we should raise an error or something if no value is provided.

---

Right, this is just a placeholder for now since I am not doing any metadata updating in this PR and it would crash otherwise since a value is needed for this. 

I can leave a note associated with issue #2272 to remove this. I think we'd catch ideally want to catch this in IM.